dm/pkg/checker/utils.go

```diff
@@ -141,3 +147,32 @@ func isMySQLError(err error, code uint16) bool {
 	e, ok := err.(*mysql.MySQLError)
 	return ok && e.Number == code
 }
+
+func genExpectGrants(privileges map[pmysql.PrivilegeType]struct{}, checkTables map[string][]string) map[pmysql.PrivilegeType]map[string]map[string]struct{} {
```

Comment:
the param checkTables is so counfunsed..... maybe wrap to a struct ?

Reply:
add some comments for it